persistence/persistence-api/src/main/java/org/cbioportal/persistence/CacheDisableSetting.java

```diff
@@ -0,0 +1,40 @@
+package org.cbioportal.persistence;
+
+import org.springframework.beans.factory.annotation.Value;
+
+public class CacheDisableSetting {
+
+    @Value("${ehcache.disable_all_caching:false}")
+    private String setting;
+
+    private String inverse = null;
+
+    private boolean booleanValue = false;
+
+    public String get() {
+        return setting;
+    }
+
+    public String getInverse() {
+        if (inverse == null) {
+            initializeOtherDataMembers();
+        }
+        return inverse;
+    }
+
+    public boolean getBoolean() {
```

Comment:
maybe also rename this to `isCachingDisabled()` or something like that which would return the value. That way I think we could get rid of these other methods in here as well. The way that the property setting and how the inverse is determined and returned is a bit more complicated that it needs to be - I recommend simplifying.

Reply:
or if the method is renamed to something like 

`isCachingEnabled()` then the return statement could be
```
return !ehcacheDisabled
```

Reply:
I now have two access functions:
- getEnabled()  (returns "true" or "false")
- isEnabled() (returns true or false)